metaflow/tutorials/06-statistics-redux/README.md

```diff
@@ -3,42 +3,20 @@
 **This example revisits 'Episode 02-statistics: Is this Data Science?'. With
 Metaflow, you don't need to make any code changes to scale-up your flow by
 running on remote compute. In this example we re-run the 'stats.py' workflow
-adding the '--with batch' command line argument. This instructs Metaflow to run
-all your steps on AWS batch without changing any code. You can control the
+adding the '--with kubernetes' command line argument. This instructs Metaflow to run
+all your steps on AWS Kubernetes without changing any code. You can control the
 behavior with additional arguments, like '--max-workers'. For this example,
 'max-workers' is used to limit the number of parallel genre-specific statistics
 computations.
 You can then access the data artifacts (even the local CSV file) from anywhere
-because the data is being stored in AWS S3.
-This tutorial uses `pandas` which may not be available in your environment.
-Use the 'conda' package manager with the `conda-forge` channel added to run
-this tutorial in any environment**
+because the data is being stored in AWS S3.**
 
 #### Showcasing:
-- '--with batch' command line option
+- '--with kubernetes' command line option
 - '--max-workers' command line option
 - Accessing data locally or remotely
 - Metaflow's conda based dependency management.
```

Comment:
tutorial 02 does not depend on conda anymore so this is not showcasing conda dependency management anymore, or am I missing something?

Reply:
correct! updating 🙏